nemo_text_processing/text_normalization/hi/data/whitelist/symbol.tsv

```diff
@@ -0,0 +1,23 @@
+&	and
```

Comment:
is this necessary?

Reply:
I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?